Doc/library/test.rst

```diff
@@ -249,7 +265,11 @@ The :mod:`test.support` module defines the following constants:
 
 .. data:: FS_NONASCII
 
-   A non-ASCII character encodable by :func:`os.fsencode`.
+   A non-ASCII character encodable by :func:`os.fsencode`.  The function
```

Comment:
"The function" may be a bit opaque here. Maybe replace `The function` by `Choosen at import time` (which breaks the rest of the sentence)?